languages/de.ini

```diff
@@ -1571,9 +1587,11 @@ VHS = "VHS"
 Video = "Video"
 Video Clips = "Videoclips"
 Video Game = "Videospiel"
+Videodisc = "Videodisk"
 Videos = "Videos"
 View Book Bag = "Zwischenablage ansehen"
 View Complete Issue = "Vollständige Ausgabe ansehen"
+View Expression of Concern Notice = "Hinweis 'Expression of Concern' anzeigen"
```

Comment:
Hier würde ich einen Doppelpunkt einfügen:
`View Expression of Concern Notice = "Hinweis: 'Expression of Concern' anzeigen"`

Reply:
The notice is called "Expression of Concern". Therefore adding a colon looks wrong for me.